source/clear-linux/tutorials/multi-boot/multi-boot.rst

```diff
@@ -3,19 +3,29 @@
 Multi-boot |CL-ATTR| with other operating systems
```

Comment:
Update title to not use Clear Linux OS e.g. "Multi-boot with other operating systems"